packages/lib/src/interactions/box.test.ts

```diff
@@ -0,0 +1,203 @@
+import { Vector3 } from 'three';
+
+import Box from './box';
+
+const ZERO_COORDS = { x: 0, y: 0, z: 0 };
+
+describe('size', () => {
+  it('should compute box size', () => {
+    const box1 = Box.empty();
+    expect(box1.size).toEqual({ width: 0, height: 0 });
+
+    const box2 = Box.empty(new Vector3(10, 10));
+    expect(box2.size).toEqual({ width: 0, height: 0 });
+
+    const box3 = Box.empty().expandByPoint(new Vector3(10, -5));
+    expect(box3.size).toEqual({ width: 10, height: 5 });
+  });
+});
+
+describe('center', () => {
+  it('should compute box center', () => {
+    const box1 = Box.empty();
+    expect(box1.center).toMatchObject(ZERO_COORDS);
+
+    const box2 = Box.empty(new Vector3(10, 10));
+    expect(box2.center).toMatchObject({ x: 10, y: 10, z: 0 });
+
+    const box3 = Box.empty().expandByPoint(new Vector3(10, -5));
+    expect(box3.center).toMatchObject({ x: 5, y: -2.5, z: 0 });
+  });
+});
+
+describe('empty', () => {
+  it('should create empty box at (0, 0, 0)', () => {
```

Comment:
```suggestion
  it('should create an empty box at (0, 0, 0)', () => {
```

Reply:
I try to avoid `a`, `an`, `the`, etc. in test labels to keep them short. This is the writing style that is used in most of our tests at the moment.